graf3d/eve7/inc/ROOT/REveFrameBox.hxx

```diff
@@ -0,0 +1,96 @@
+// @(#)root/eve:$Id$
+// Authors: Matevz Tadel & Alja Mrak-Tadel: 2006, 2007
+
+/*************************************************************************
+ * Copyright (C) 1995-2007, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT_REveFrameBox
+#define ROOT_REveFrameBox
+
+#include "ROOT/REveUtil.hxx"
+#include "TObject.h"
+
+namespace ROOT {
+namespace Experimental {
+
+class REveFrameBox : public TObject, public REveRefBackPtr
+{
+   friend class REveFrameBoxGL;
+
+public:
+   enum EFrameType_e  { kFT_None, kFT_Quad, kFT_Box };
+
+private:
+   REveFrameBox(const REveFrameBox&);            // Not implemented
+   REveFrameBox& operator=(const REveFrameBox&); // Not implemented
+
+protected:
+   EFrameType_e fFrameType;
+   Int_t        fFrameSize;
+   Float_t     *fFramePoints;  //[fFrameSize]
```

Comment:
Can one use here std::vector<Float_t> instead?
If you need access to plain array, you always can use `vector::data()` method